bin/nbb

```diff
@@ -344,6 +339,24 @@ find_centos_7_ami_ids() {
   done
 }
 
+shutdown_instances() {
+  aws ec2 describe-instances --filters "Name=instance-state-name,Values=running" | jq -c -r '.Reservations[] .Instances[] .InstanceId' > instance_list
+  echo "You are about to terminate the following instances:"
+  cat instance_list
+  read -r -p "Are you sure? [y/N] " response
+  if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]
+  then
+    IFS=$'\n'  # make newlines the separator
+    for i in `cat ./instance_list`; do
+      echo "Shutting down $i"
+      aws ec2 terminate-instances --instance-ids $i
```

Comment:
Er, no. I added this to the trello card last week, but it wasn't read.

You can see what `make destroy` does before this patch.  It logs into the bastion host, runs a provision destroy (which has stopped working for some reason) and then calls terraform to destroy the rest.

Terraform only destroys stuff that it manages.  Since it does't manage bosh (that's handled by bosh-init) or any of the stuff in cloud foundry (that gets managed by bosh during the deploy) it doesn't know to destroy them.

This is a case of having two different configurations for AWS stuff - the terraform component for the VPC, and the bosh component.  The bosh component isn't working correctly, which causes the terraform component to not work correctly (since there are instances that terraform doesn't manage on VPC subnets that it does manage)